README.md

```diff
@@ -1 +1,49 @@
-# fuel-v2-contracts
\ No newline at end of file
+# Fuel Solidity Contracts
+
+<!-- Disable markdownlint for long lines. -->
+<!-- markdownlint-disable-file MD013 -->
+
+![ci](https://github.com/fuellabs/fuel-v2-contracts/workflows/Node.js%20Tests%20and%20Coverage/badge.svg?branch=master)
```

Comment:
Icon is broken when I preview

Reply:
Because the workflow isn't even part of the repo yet ;)